pkg/ingestor/parser/spdx/parse_spdx.go

```diff
@@ -166,6 +189,13 @@ func (s *spdxParser) getPackageElement(elementID string) []*model.PkgInputSpec {
 	return nil
 }
 
+func (s *spdxParser) getTopLevelPackageElement(elementID string) []*model.PkgInputSpec {
+	if packNode, ok := s.topLevelPackages[string(elementID)]; ok {
+		return packNode
+	}
+	return nil
```

Comment:
This should be an error message. As if there is no top-level package, we should not proceed. We are doing this in CDX as well: https://github.com/guacsec/guac/blob/165ec24a416eac6ba94924982a939e3954710235/pkg/ingestor/parser/cyclonedx/parser_cyclonedx.go#L110-L112

Reply:
This is for the name case in SPDX Syft, i think we still want to have the same behavior?

Reply:
Ok, since GetPredicates does not return an error I logged an error message and returned empty predicates.

This code path isn't actually reachable today, but I didn't want to put a panic because if something changes in the future and GUAC parses a bad SBOM (likely) it would crash guac